mimic/test/test_images.py

```diff
@@ -0,0 +1,113 @@
+"""
+Tests for :mod:`nova_api` and :mod:`nova_objects` for images.
+"""
```

Comment:
@lekhajee Can we just pull all the nova images tests into this file and remove them from test_nova.py since it is really big and it will start to address issue https://github.com/rackerlabs/mimic/issues/414